isleconfig.py

```diff
@@ -17,9 +17,9 @@
                        "norm_profit_markup": 0.15,
                        "rein_norm_profit_markup": 0.15,
                        "dividend_share_of_profits": 0.4,
-                       "mean_contract_runtime": 36,
-                       "contract_runtime_halfspread": 10,
-                       "default_contract_payment_period": 12,
+                       "mean_contract_runtime": 12,
+                       "contract_runtime_halfspread": 2,
+                       "default_contract_payment_period": 3,
```

Comment:
I'm fine with going down to quarterly payments. But I do not quite see what effect this will have.